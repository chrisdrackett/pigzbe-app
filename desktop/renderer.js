import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
