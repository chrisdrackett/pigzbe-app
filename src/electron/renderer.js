import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
