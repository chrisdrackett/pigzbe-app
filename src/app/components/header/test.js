import React from 'react';
import renderer from 'react-test-renderer';
import Header from './';

const props = {
    showSettings: false,
    navigation: null
};

describe('Header', () => {
    test('renders correctly', () => {
        renderer.create(<Header {...props}/>);
    });
});
