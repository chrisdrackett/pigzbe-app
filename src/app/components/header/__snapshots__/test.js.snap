// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#003278",
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/images/pigzbe_logo_inapp.png",
      }
    }
    style={
      Object {
        "height": 25,
        "marginBottom": 10,
        "marginTop": 34,
        "width": 65,
      }
    }
  />
</View>
`;
